wemake_python_styleguide/visitors/ast/imports.py

```diff
@@ -39,30 +40,18 @@ def __init__(
         self._error_callback = error_callback
         self._options = options
 
-    def check_nested_import(self, node: AnyImport) -> None:
+    @abc.abstractmethod
+    def validate(self, node: AnyImport):
```

Comment:
`-> None`

Reply:
We can just call this method differently. Like `_validate_any_import` and then call it in specific `.validate` method instead of `super().validate()`

Reply:
Sure, but then we need to resign from declaring `.validate()` as abstractmethod and just provide this method in 2 concrete validators to satisfy mypy, is that what you suggest?

Reply:
Yes 👍